### ISSUE TITLE
Support single value metric

### ISSUE THREAD

**Issue by uduse** [aimhubio/aim#1528]
### Motivation

There are only two scalar metrics matter for my run, and both of them only need one single value (no steps or epochs).

![image](https://user-images.githubusercontent.com/4717005/158484454-fbef76c9-5b2f-4e2f-a62f-031a99c7d22e.png)

They are hard to observe in the metrics tab, and I can't even copy the result values.

### Pitch

Render single step values differently, also add a button to copy the value or something

**Comment by mihran113** [comment]
Hey @uduse. Won't setting them as a parameter for run will do the trick? (i.e `run['return_geomean'] = some_value`). That should solve the issue with copying the result values and you can visualize them using the params explorer.

**Comment by uduse** [comment]
Yes that would do the trick but it's still a trick. I think these are metrics and they deserved to be treated like metrics.

**Comment by roubkar** [comment]
Hey @uduse, you can see the last values of your metrics in the `Overview` tab of `Single Run Page`.

<img width="1123" alt="Screen Shot 2022-03-16 at 09 52 07" src="https://user-images.githubusercontent.com/9951653/158525301-8c0cedfe-153a-49b3-ba99-5f659ea9f2c1.png">

**Comment by uduse** [comment]
@roubkar Nice new feature!
